src/solver/postprocess.jl

```diff
@@ -122,3 +135,33 @@ function postprocess_root(x)
     end
     x # unreachable
 end
+
+function check_trig_consts(x)
+    !iscall(x) && return x
+
+    oper = operation(x)
+    inv_opers = [asin, acos, atan]
+    inv_exacts = [0, Symbolics.term(*, pi),
+        Symbolics.term(/,pi,3),
+        Symbolics.term(/, pi, 2),
+        Symbolics.term(/, Symbolics.term(*, 2, pi), 3),
+        Symbolics.term(/, pi, 6),
+        Symbolics.term(/, Symbolics.term(*, 5, pi), 6),
+        Symbolics.term(/, pi, 4)
+    ]
+
+    if any(isequal(oper, o) for o in inv_opers) && isempty(Symbolics.get_variables(x))
+        val = Symbolics.symbolic_to_float(x)
+        for i in eachindex(inv_exacts)
+            exact_val = Symbolics.symbolic_to_float(inv_exacts[i])
+            if isapprox(exact_val, val, atol=1e-6)
+                return inv_exacts[i]
+            elseif isapprox(-exact_val, val, atol=1e-6)
```

Comment:
```suggestion
            if isapprox(exact_val, val)
                return inv_exacts[i]
            elseif isapprox(-exact_val, val)
```

`sqrt(eps)` is just a better default